src/components/victory-axis.jsx

```diff
@@ -193,8 +193,10 @@ class VAxis extends React.Component {
   }
 
   getTickFormat(props) {
-    if (props.tickFormat) {
+    if (props.tickFormat && _.isFunction(props.tickFormat)) {
       return props.tickFormat;
+    } else if (props.tickFormat && _.isArray(props.tickFormat)) {
+      return (x) => props.tickFormat[_.indexOf(this.ticks, x)];
```

Comment:
I anticipate this `indexOf` causing us some problems either due to duplicate `x` values (which, sure, might not be allowed anyway, but it shouldn't cause *this* to fail) or for performance (searching an array with `indexOf` over and over).

Better might be to update anywhere we call `tickFormat` and send it both `x` and the `index`, just like Array `map` does, so here we can do `return (x, i) => props.tickFormat[i]`.